frontend/src/components/search/Search.tsx

```diff
@@ -71,7 +96,50 @@ export default function Search(props: { isOpen: boolean, closeModal: () => void
 				<span className={ "caption" }>{ t('dashboard.search.info') }</span>
 			</p>
 			
-			<p>{ suggestions.length } suggestions.</p>
+			{ suggestions.map((suggestion, i) => <div key={ i } className={ "suggestion" } tabIndex={ 0 }>
+				<div className={ "suggestion-main" }>
+					<div className={ "suggestion-title" }>
+						{ suggestion.isDiscussion
+							? <span className={ "badge badge-outline" }>
+									<i className={ "fi fi-rr-comments-question" }
+									   style={ { marginRight: "calc(var(--spacing) / 2)" } }/>
+									Discussion
+							</span>
+							: <span className={ "badge badge-outline" }>
+									<i className={ "fi fi-rr-interrogation" }
+									   style={ { marginRight: "calc(var(--spacing) / 2)" } }/>
+									Question
```

Comment:
Due to i18n: Extract hardcoded text into translation file, allowing for easier translation and maintenance of multilingual requirement.